### ISSUE TITLE
commands not showing with white cmd background

### ISSUE THREAD

**Issue by danthe1st** [microsoft/winget-cli#723]
# Brief description of your issue

If the background of cmd is white, the subcommands are not visible.

# Steps to reproduce

* set the background of cmd to white and the foreground to black in the `Properties` of the cmd window
![image](https://user-images.githubusercontent.com/34687786/106145542-f8501d80-6175-11eb-82a5-3dd1d569abb6.png)
![image](https://user-images.githubusercontent.com/34687786/106145566-ff772b80-6175-11eb-9f1b-d770c0a7f4b0.png)
* enter `winget`

# Expected behavior
The subcommands listed in the should have the same color as the background.
![image](https://user-images.githubusercontent.com/34687786/106146314-d2774880-6176-11eb-8800-77c58cd9f4fa.png)

# Actual behavior

The subcommands listed in the help have the same color as the background.

![image](https://user-images.githubusercontent.com/34687786/106146467-fcc90600-6176-11eb-9167-c154ac3ccb9c.png)

# Environment

```none
[winget --info]
Windows Package Manager v0.2.3162 Preview
Microsoft Windows [Version 10.0.19041.388]
Package: Github Release v0.2.3162-preview
```

**Comment by raspher** [comment]
Please provide screenshot with text selected (maybe it will be readable)
![image](https://user-images.githubusercontent.com/23345803/110019342-a415fb80-7d28-11eb-883e-86934d6fc760.png)

**Comment by danthe1st** [comment]
It is not readable when selected:
![image](https://user-images.githubusercontent.com/34687786/110076743-4a8fea00-7d85-11eb-81d8-d22242fdf9e0.png)
